### ISSUE TITLE
Request for MS-ASL Dataset

### ISSUE THREAD

**Issue by wildhorse228** [0hyunU/MS-ASL-Video-Dataset-Downloader#1]
Although MS-ASL provided a link, cannot be downloaded. Can the source file be provided for download? I would greatly appreciate it!

**Comment by zrohyun** [comment]
When you say source file, do you mean the original video file?

**Comment by wildhorse228** [comment]
> 当您说源文件时，您指的是原始视频文件吗？

Yes. Would it be possible for you to provide a download link for the complete dataset? I would greatly appreciate it!

**Comment by zrohyun** [comment]
I'm sorry, but this project concluded quite a long time ago, and I no longer have the dataset.
